restricted_QAQC/global.R

```diff
@@ -154,19 +156,27 @@ standardizeClasses <- function(d) {
    # ColClasses :    vector of desired class types for the data.frame
    # message(paste("In standardizeClasses for", deparse(substitute(d))))
    r <- nrow(d)
+   d<-d %>% select(-year)
```

Comment:
this function generally doesn't need to be modified. I'm nervous that changing classes and columns in here will cause issues elsewhere.